build-tools/debian-metadata/xamarin.android.install

```diff
@@ -0,0 +1,2 @@
+bin/Debug/bin/xabuild		/usr/bin
```

Comment:
Well, yes, since we're building the whole debian/ folder as part of a makefile before trying to use it, we can do anything we like.